include/mbedtls/config.h

```diff
@@ -484,14 +484,14 @@
 /**
  * \def MBEDTLS_MD2_PROCESS_ALT
  *
- * MBEDTLS__FUNCTION_NAME__ALT: Uncomment a macro to let mbed TLS use you
+ * MBEDTLS__FUNCTION_NAME__ALT: Uncomment a macro to let Mbed TLS use you
```

Comment:
```suggestion
 * MBEDTLS__FUNCTION_NAME__ALT: Uncomment a macro to let Mbed TLS use an
```

I guess this was originally supposed to be "your", although probably "an" is better?

Reply:
There are several "your"s, and while this is a pre-existing typo that would be easy to fix here, it might be better to hold off and have a separate PR to change all the "your"s (and this "you") to be third-person/impersonal

Reply:
This is fixing a bug in the release process (we missed one occurrence of the version number due to the capitalization inconsistency), so I'd rather not increase the scope.